blog/electron-27-0.md

```diff
@@ -0,0 +1,152 @@
+
+The `ipcRenderer.sendTo()` API has been deprecated. It should be replaced by setting up a [`MessageChannel`](tutorial/message-ports.md#setting-up-a-messagechannel-between-two-renderers) between the renderers.
+
+The `senderId` and `senderIsMainFrame` properties of `IpcRendererEvent` have been deprecated as well.
+
+### Removed: color scheme events in `systemPreferences`
+
+The following `systemPreferences` events have been removed:
+
+- `inverted-color-scheme-changed`
+- `high-contrast-color-scheme-changed`
+
+Use the new `updated` event on the `nativeTheme` module instead.
+
+```js
+// Removed
+systemPreferences.on('inverted-color-scheme-changed', () => {
+  /* ... */
+});
+systemPreferences.on('high-contrast-color-scheme-changed', () => {
+  /* ... */
+});
+
+// Replace with
+nativeTheme.on('updated', () => {
+  /* ... */
+});
+```
+
+### Removed: `webContents.getPrinters`
+
+The `webContents.getPrinters` method has been removed. Use
+`webContents.getPrintersAsync` instead.
+
+```js
+const w = new BrowserWindow({ show: false });
+
+// Removed
+console.log(w.webContents.getPrinters());
+// Replace with
+w.webContents.getPrintersAsync().then((printers) => {
+  console.log(printers);
+});
+```
+
+### Removed: `systemPreferences.{get,set}AppLevelAppearance` and `systemPreferences.appLevelAppearance`
+
+The `systemPreferences.getAppLevelAppearance` and `systemPreferences.setAppLevelAppearance`
+methods have been removed, as well as the `systemPreferences.appLevelAppearance` property.
+Use the `nativeTheme` module instead.
+
+```js
+// Removed
+systemPreferences.getAppLevelAppearance();
+// Replace with
+nativeTheme.shouldUseDarkColors;
+
+// Removed
+systemPreferences.appLevelAppearance;
+// Replace with
+nativeTheme.shouldUseDarkColors;
+
+// Removed
+systemPreferences.setAppLevelAppearance('dark');
+// Replace with
+nativeTheme.themeSource = 'dark';
+```
+
+### Removed: `alternate-selected-control-text` value for `systemPreferences.getColor`
+
+The `alternate-selected-control-text` value for `systemPreferences.getColor`
+has been removed. Use `selected-content-background` instead.
+
+```js
+// Removed
+systemPreferences.getColor('alternate-selected-control-text');
+// Replace with
+systemPreferences.getColor('selected-content-background');
+```
+
+### New Features
+
+- Added app accessibility transparency settings api [#39631](https://github.com/electron/electron/pull/39631)
+- Added support for `chrome.scripting` extension APIs [#39675](https://github.com/electron/electron/pull/39675)
+- Enabled `WaylandWindowDecorations` by default [#39644](https://github.com/electron/electron/pull/39644)
+
+## End of Support for 24.x.y
+
+Electron 24.x.y has reached end-of-support as per the project's [support policy](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#version-support-policy). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+| E27 (Aug'23) | E28 (Oct'23) | E29 (Jan'24) |
```

Comment:
I think the dates in this header need to be updated.

Reply:
D'oh they sure do, thanks for the catch, will update those